features/provider_due_amount.feature

```diff
@@ -13,3 +13,17 @@ Examples:
 | 0               | 0             | 0          |
 
 
+
+Scenario Outline: Payment type breakdown
+Given the account has a balance of <balance>
+When payment of <due amount> is due
+Then a levy payment of <levy amount> is made
+And a government payment of <government amount> is made
+And a employer payment of <employer amount> is expected
+Examples:
+| balance | due amount | levy amount | government amount | employer amount |
```

Comment:
Wordings are as per the story DPP-421 so if anything needs changing, can you please update the story and I'll update the specs.